docs/Documentation/UserGuide.md

```diff
@@ -0,0 +1,24 @@
+# RBM Model User Guide
```

Comment:
This page needs some work - it looks like you just copied over from the VIC page - which is fine, that's what I did.  But it should be updated to match the workflow of RBM (VIC -> RVIC -> RBM).